### PR TITLE
Create Possible Index - Change new person card UI

### PR DESCRIPTION
**Pull request by atoncetti**
Parial implementation of [#333](https://github.com/jshmrtn/hygeia/issues/333#issuecomment-982561095) (comment)

Tests are expected to fail.

### CODE REVIEW

**Comment by maennchen** [comment]
## Testing

* Append new people on bottom, not top
* suggestions should only be shown, if the form has any changes
* contact method value looses focus while typing
* create form BG
* saving error:
```
[error] GenServer #PID<0.3186.0> terminating
** (ArgumentError) you provided a changeset with an action already set to :validate when trying to insert it
    (ecto 3.7.1) lib/ecto/multi.ex:413: Ecto.Multi.put_action/2
    (ecto 3.7.1) lib/ecto/multi.ex:401: Ecto.Multi.add_changeset/5
    (hygeia 1.34.0-beta.12) lib/hygeia/helpers/versioning.ex:84: Hygeia.Helpers.Versioning.run_ecto_multi/2
    (hygeia 1.34.0-beta.12) lib/hygeia/case_context.ex:296: Hygeia.CaseContext.create_person/1
    (hygeia 1.34.0-beta.12) lib/hygeia_web/live/case_live/create_possible_index/service.ex:32: anonymous fn/2 in HygeiaWeb.CaseLive.CreatePossibleIndex.Service.upsert/1
    (elixir 1.12.3) lib/enum.ex:1582: Enum."-map/2-lists^map/1-0-"/2
    (ecto_sql 3.7.1) lib/ecto/adapters/sql.ex:1021: anonymous fn/3 in Ecto.Adapters.SQL.checkout_or_transaction/4
    (db_connection 2.4.1) lib/db_connection.ex:1531: DBConnection.run_transaction/4
    (hygeia 1.34.0-beta.12) lib/hygeia_web/live/case_live/create_possible_index/service.ex:22: HygeiaWeb.CaseLive.CreatePossibleIndex.Service.upsert/1
    (hygeia 1.34.0-beta.12) lib/hygeia_web/live/case_live/create_possible_index.ex:112: HygeiaWeb.CaseLive.CreatePossibleIndex.save/1
    (hygeia 1.34.0-beta.12) lib/hygeia_web/live/case_live/create_possible_index.ex:152: HygeiaWeb.CaseLive.CreatePossibleIndex.handle_info/2
    (phoenix_live_view 0.16.4) lib/phoenix_live_view/channel.ex:250: Phoenix.LiveView.Channel.handle_info/2
    (stdlib 3.16.1) gen_server.erl:695: :gen_server.try_dispatch/4
    (stdlib 3.16.1) gen_server.erl:771: :gen_server.handle_msg/6
    (stdlib 3.16.1) proc_lib.erl:226: :proc_lib.init_p_do_apply/3
Last message: :proceed
 ```

File: lib/hygeia_web/live/case_live/create_possible_index/form_step/define_people.ex [hunk 1]
```diff
@@ -135,171 +80,62 @@ defmodule HygeiaWeb.CaseLive.CreatePossibleIndex.FormStep.DefinePeople do
...
-     |> assign(:suggestions, suggestions)}
-  end
-
-  @impl Phoenix.LiveComponent
-  def handle_event("clear_search", _params, socket) do
-    {:noreply,
-     socket
-     |> clear_search()
-     |> clear_suggestions()}
+  def handle_event("clear_person", _params, socket) do
```
Comment (maennchen) [inline lib/hygeia_web/live/case_live/create_possible_index/form_step/define_people.ex:105]: I can't see that event being triggered anywhere

Comment (atoncetti) [reply lib/hygeia_web/live/case_live/create_possible_index/form_step/define_people.ex:105]: solved thanks

File: lib/hygeia_web/live/person_live/create_form.sface [hunk 2]
```diff
@@ -165,8 +165,8 @@
         </div>
       </div>
     </div>
-    {#if slot_assigned?(:footer)}
-      <#slot name="footer" />
-    {/if}
   </div>
+  {#if slot_assigned?(:footer)}
```
Comment (maennchen) [inline lib/hygeia_web/live/person_live/create_form.sface:169]: This change breaks the styling of person create. Needs a quick fix.

Comment (atoncetti) [reply lib/hygeia_web/live/person_live/create_form.sface:169]: Fixed

**Review by maennchen — DISMISSED** [review]
code seems fine, now checking functionality again :+1:

**Comment by maennchen** [comment]
@atoncetti One small issue:

* [ ] When accepting a submission and the define people step is immediately complete & valid, the next button is disabled. changing any field will enable it

**Review by maennchen — APPROVED** [review]

**Pull request merged**